oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/storage/model/ModelName.java

```diff
@@ -30,10 +43,73 @@ public static String build(Downsampling downsampling, String modelName) {
                 return modelName + Const.ID_SPLIT + Downsampling.Day.getName();
             case Hour:
                 return modelName + Const.ID_SPLIT + Downsampling.Hour.getName();
-            //            case Second:
-            //                return modelName + Const.ID_SPLIT + Downsampling.Second.getName();
             default:
                 return modelName;
         }
     }
+
+    public static String[] build(Downsampling downsampling, String modelName, long startTB, long endTB) {
```

Comment:
This build method is for ES storage only, please move inside the plugin.

Reply:
ok